test-utils/state-viewer/src/main.rs

```diff
@@ -585,6 +611,30 @@ fn main() {
                 account_id
             );
         }
+        ("dump_account_storage", Some(args)) => {
+            let account_id = args.value_of("account").expect("account is required");
```

Comment:
It's better to mark clap arg as required and just unwrap here: https://docs.rs/clap/2.33.3/clap/struct.Arg.html#method.required